src/main/java/io/aiven/kafka/connect/common/config/FormatterUtils.java

```diff
@@ -1,57 +0,0 @@
-/*
```

Comment:
@ivanyu I'm afraid this class is still in-use in S3 for old configs https://github.com/aiven/aiven-kafka-connect-s3/blob/905826ed1eeb2bc39e4dc923473231ec9a66db4e/src/main/java/io/aiven/kafka/connect/s3/S3SinkTask.java#L197

Reply:
Valid point
I'm changing in the next pr into S3, sorry, should have mentioned